includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -41,13 +41,6 @@ class WC_Stripe_UPE_Payment_Gateway extends WC_Gateway_Stripe {
 		WC_Stripe_UPE_Payment_Method_Cash_App_Pay::class,
 	];
 
-	/**
-	 * Stripe intents that are treated as successfully created.
-	 *
-	 * @type array
-	 */
-	const SUCCESSFUL_INTENT_STATUS = [ 'succeeded', 'requires_capture', 'processing' ];
```

Comment:
Since this was publicly available, do we need to deprecate this before removing it? Anyone attempting to use `WC_Stripe_UPE_Payment_Gateway::SUCCESSFUL_INTENT_STATUS` will throw a PHP fatal error after upgrading.

Reply:
Good point. Deprecated it instead of removing it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3648/commits/9f3895ac742aa8f0b23b36a3e7a63a7f3f0c09b1